app/(navigation)/themes/themes/rubinho/nightrose.json

```
@@ -0,0 +1,21 @@
+{
+  "author": "Rúben Gomes",
+  "authorUsername": "rubinho",
+  "version": "1",
+  "name": "Nigth Rose",
```

```suggestion
  "name": "Night Rose",
```

Is this a typo or on purpose?